glue/flagging_script_glue/flagging.py

```diff
@@ -849,15 +849,15 @@ def outlier_type(df: pd.DataFrame, condos: bool) -> pd.DataFrame:
             "Home flip sale (high)",
             "Family sale (high)",
             "Non-person sale (high)",
-            "Anomaly (high)",
+            # "Anomaly (high)",
             "High price swing",
             "High price (raw & sqft)",
             "High price (raw)",
             "High price (sqft)",
             "Home flip sale (low)",
             "Family sale (low)",
             "Non-person sale (low)",
-            "Anomaly (low)",
+            # "Anomaly (low)",
```

Comment:
The commented out iso_forest code is just to enable faster iteration since it is the longest part of the flagging proccess.